runtime/peaq-dev/src/lib.rs

```diff
@@ -711,6 +711,9 @@ parameter_types! {
 	pub const PotStakeId: PalletId = PalletId(*b"PotStake");
 	pub const PotMorId: PalletId = PalletId(*b"PotMchOw");
 	pub const PotTreasuryId: PalletId = TreasuryPalletId::get();
+	pub const PotCoretimeId: PalletId = PalletId(*b"PotCoret");
+	pub const PotSubsidizationId: PalletId = PalletId(*b"PotSubsi");
+	pub const PotDepinStakingId: PalletId = PalletId(*b"PotDPStk");
```

Comment:
Why don't we have PotDepinIncentivisationId in the peaq-dev? We should have to distribute the token to PotDepinIncentivisationId and MorPallet. Let us have a discuss